### PR TITLE
prepare v10.0.0-beta.10

### PR DESCRIPTION
**Pull request by blagoev**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "realm",
   "description": "Realm is a mobile database: an alternative to SQLite and key-value stores",
-  "version": "10.0.0-beta.10",
+  "version": "10.0.0-beta.11",
```
Comment (steffenagger) [inline package.json:4]: `10.0.0-beta.9` is the latest beta release on https://www.npmjs.com/package/realm & changelog marks updated as `10.0.0-beta.10`

File: dependencies.list [hunk 2]
```diff
@@ -1,4 +1,4 @@
 PACKAGE_NAME=realm-js
-VERSION=10.0.0-beta.10
-REALM_CORE_VERSION=10.0.0-beta.4
-REALM_SYNC_VERSION=10.0.0-beta.6
+VERSION=10.0.0-beta.11
```
Comment (steffenagger) [inline dependencies.list:2]: `10.0.0-beta.9` is the latest beta release on https://www.npmjs.com/package/realm & changelog marks updated as `10.0.0-beta.10`

**Review by steffenagger — CHANGES_REQUESTED** [review]
See inline comments

File: package.json (hunk 1 above)
Comment (blagoev) [reply package.json:4]: that's a mistake see the branch name :)

**Review by steffenagger — APPROVED** [review]
LGTM

**Review by bmunkholm — APPROVED** [review]

**Review by kneth — APPROVED** [review]

**Pull request merged**